### PR TITLE
[CHANGELOG] v4.0.0-alpha-23

### PR DESCRIPTION
**Pull request by DanailH**
CHANGELOG changes for the `v4.0.0-alpha-23` release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,40 @@
...
+
+Big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇵🇱 Added plPL locale (#1117) @LarsKumbier
+- 🎁 Add onScrollBottom to support infinite loading (#1199) @DanailH
+- 🕹 Add shift as multiple key (#1203) @dtassone
+- ⚡️ Edit cell accessibility (#1205) @dtassone
+- 🐞 Bugfixes
+
+### @material-ui/x-grid@v4.0.0-alpha.22 / @material-ui/data-grid@v4.0.0-alpha.22
```
Comment (oliviertassinari) [inline CHANGELOG.md:18]:
```suggestion
### @material-ui/x-grid@v4.0.0-alpha.23 / @material-ui/data-grid@v4.0.0-alpha.23
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,40 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.23](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.22...v4.0.0-alpha.23)
+
+_Mar 22, 2021_
+
+Big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇵🇱 Added plPL locale (#1117) @LarsKumbier
+- 🎁 Add onScrollBottom to support infinite loading (#1199) @DanailH
```
Comment (oliviertassinari) [inline CHANGELOG.md:13]: This seems to be the biggest improvement of the release, we might want to mention it once, add a link to the documentation/record a video to demonstrate what it does.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,40 @@
...
+- 🕹 Add shift as multiple key (#1203) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:14]: I think that we should mention that it's for the multi-sort feature, it's not clear otherwise.

Comment (DanailH) [reply CHANGELOG.md]: Updated

File: CHANGELOG.md (hunk 2 above)
Comment (DanailH) [reply CHANGELOG.md]: Updated. Added additional explanation and a link to the docs.

**Review by oliviertassinari — APPROVED** [review]

**Comment by oliviertassinari** [comment]
We forgot to mention this breaking change https://github.com/mui-org/material-ui-x/pull/1158

**Pull request merged**